workflow/scripts/predict_immunogenicity.py

```
@@ -4,15 +4,11 @@
-    for entry in fh:
-        entry_splitted = entry.rstrip().split('\t')
-        im_fh.write(f'{entry_splitted[15]}\n')
-    im_fh.close()
-    fh.close()
+    with open(sys.argv[1], 'r') as fh, open('workflow/scripts/im.txt', 'w') as im_fh:
+        next(fh)
+        for entry in fh:
+            entry_splitted = entry.rstrip().split('\t')
+            im_fh.write(f'{entry_splitted[15]}\n')
 
     immunogenecity_values = ['immunogenicity'] + calc_immunogenecity('workflow/scripts/im.txt')
```

Ruff (0.15.13)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)